### DISCUSSION TITLE
any point in A incapable cameras?

### DISCUSSION THREAD

**Discussion by uglyfish333** [blakeblackshear/frigate#7309] [Ask A Question] [answered]
Hey there. Planning on putting together a few security cameras with my home media/assistant server, and I'm noticing that cameras with built in AI function tend to get recommended. I was considering something like the Dahua T5442T-ZE to run with my server and a USB coral module, but I'm wondering if I'm overpaying for AI that wouldn't actually be used with a frigate setup. Like if the software is handling the motion and object detection, recording, and pushing notifications/triggers to home assistant..is all the native AI on the camera going unused? Should I just save money and buy something with a similar sensor, etc and let Frigate handle all the advanced functions?

**Comment by NickM-27** [comment, marked as answer]
There is overlap between cameras with AI built in and the recommendations, but that is because the cameras that have AI built in also generally have important features like 3 customizable streams, smarter WDR, IR lighting, etc. and other things. 

> I was considering something like the Dahua T5442T-ZE 

I have this camera and quite like it

**Reply by uglyfish333** [reply]
Good to know! So i take it that there not really a non-AI version with similar specs for a better price. All the cheaper non-AI versions I'm seeing on ali have much smaller sensors (1/3 instead of 1/1.8) so probably perform a lot worse in low light, and are all fixed instead of variable focus. That's just looking at Dahua branded items though.

I guess the move is to just suck it up and paid for unused features. Would you say that the AI frigate performs is better than the on board? Or any ability to share the load so to speak and leverage some of the built in functionality?

**Reply by NickM-27** [reply]
I think the AI is a bit different, has some cool features but less customizable and much less compatible with normal platforms like HA or MQTT.

**Reply by uglyfish333** [reply]
Sorry when you say the AI is a bit different and less compatible, do you mean the AI on camera or in frigate? Sorry for all the questions, really appreciate the input.

**Reply by NickM-27** [reply]
I mean the cameras AI is less compatible
